src/main/resources/assets/tconstruct/book/materials_and_you/en_us/tools/mattock.json

```diff
@@ -2,18 +2,17 @@
   "tool": "tconstruct:mattock",
   "text": [
     {
-      "text": "The Cutter Mattock is a versatile farming tool. It is effective on wood, dirt, and plants."
+      "text": "The Mattock is a versatile chopping tool. It is effective on wood, dirt, and plants."
```

Comment:
It is not effective on plants, not in the common sense. It works on dirt and wood. Calling it a chopping tool sounds a little weird when its our primary shovel, but I guess we can say we chop dirt...